src/app/cli/src/transition_frontier_persistence_rpc_worker.ml

```diff
@@ -1,103 +0,0 @@
-open Core
-open Async
```

Comment:
Was this file never used?

Reply:
It was used to test something at one point, but I checked with @wu-s-john and he told me I could delete it. It's outdated and didn't compile. It wasn't being compiled before since it was in a package that only had a binary target, meaning that dune will only build deps in that package that are actually needed in the binary. When this was split out to a lib, dune would always attempt to build it.